.gitignore

```diff
@@ -4,3 +4,4 @@ kore
 *.module
 *.DSYM
 cert
+.idea
```

Comment:
That line is because I'm using CLion, and I don't want to add garbage to parent repo.